docs/installation.md

```diff
@@ -139,6 +139,32 @@ are available for Windows and Linux.
     # chmod +x watchmaker-latest-standalone-linux-x86_64
     ```
 
+## Prerequisites for features specific to AWS and Azure
+
+Watchmaker has some features specific to AWS and Azure:
+
+    * AWS:
+        *  Download files in config references from Amazon S3
```

Comment:
Extra space.

```suggestion
        * Download files in config references from Amazon S3
```

Reply:
Why your eyes gotta be so sharp? 😢 Fixed!